examples/chef/common/clusters/resource-monitoring/chef-resource-monitoring-delegates.cpp

```diff
@@ -96,25 +98,161 @@ void HepaFilterMonitoring::Shutdown()
     gHepaFilterDelegate.reset();
 }
 
+chip::Protocols::InteractionModel::Status
+ChefResourceMonitorInstance::ExternalAttributeWrite(const EmberAfAttributeMetadata * attributeMetadata, uint8_t * buffer)
+{
+    Protocols::InteractionModel::Status ret = Protocols::InteractionModel::Status::Success;
+    AttributeId attributeId                 = attributeMetadata->attributeId;
+
+    switch (attributeId)
+    {
+    case HepaFilterMonitoring::Attributes::Condition::Id: {
+        uint8_t newCondition = *(uint8_t *) buffer;
+        ret                  = UpdateCondition(newCondition);
+    }
+    break;
+    case HepaFilterMonitoring::Attributes::ChangeIndication::Id: {
+        ResourceMonitoring::ChangeIndicationEnum newIndication =
+            static_cast<ResourceMonitoring::ChangeIndicationEnum>(*(uint8_t *) buffer);
+        ret = UpdateChangeIndication(newIndication);
+    }
+    break;
+    case HepaFilterMonitoring::Attributes::InPlaceIndicator::Id: {
+        bool newInPlaceIndicator = *(bool *) buffer;
+        ret                      = UpdateInPlaceIndicator(newInPlaceIndicator);
+    }
+    break;
+    case HepaFilterMonitoring::Attributes::LastChangedTime::Id: {
+        uint32_t newValue = 0;
+        uint16_t tlvLen   = *(uint16_t *) buffer;
+        chip::TLV::TLVReader reader;
+        reader.Init(buffer + sizeof(uint16_t), tlvLen);
+        reader.Next();
+        reader.Get(newValue);
+        DataModel::Nullable<uint32_t> newLastChangedTime = DataModel::MakeNullable(newValue);
+        ret                                              = UpdateLastChangedTime(newLastChangedTime);
+    }
+    break;
+    case HepaFilterMonitoring::Attributes::DegradationDirection::Id:
+    default: {
+        ChipLogError(Zcl, "Unsupported External Attribute Read: %d", static_cast<int>(attributeId));
```

Comment:
The log mentions a read but this is in the write method.

Reply:
Fixed, Thanks for catching this.